specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -6202,12 +6506,11 @@
       ],
       "properties": {
         "hookParameter": {
-          "$ref": "#/definitions/WebhookHookParameter"
+          "$ref": "#/definitions/WebhookHookParameterPatch"
```

Comment:
Should `EmailHookInfo` have an `EmailHookParameterPatch` as well? Currently it only has an `EmailHookParameter`. It's only an array, though:
https://github.com/Azure/azure-rest-api-specs/blob/e24cf486381bbff6a1e06f161dcef327dcaacb94/specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json#L6250-L6265

If we add new properties to it in the future, a patch model may be necessary. I think it wouldn't be a breaking change in this case, though. Is that right? @johanste

Reply:
Yes, it does have EmailHookParameterPatch. There was a bug in the swagger generation code. Fixed.